MAINTAINER_GUIDELINES.md

```diff
@@ -0,0 +1,11 @@
+## Maintainer Guidelines
+
+### Releasing working code
+
+* run a full suite of actual integration tests before accepting a PR
+* releases should have a full integration test run before releasing to catch stragglers
+
+### Development
+
+* Rusoto should be able to be used and developed on Rust's stable channel
```

Comment:
This one is a bit tricky due to rustfmt's nightly requirement.  See https://github.com/rusoto/rusoto/pull/897 for more details.